src/main/java/picard/arrays/VcfToAdpc.java

```diff
@@ -62,55 +68,85 @@
                     "An adpc.bin file is a binary file containing genotyping array intensity data that can be exported " +
                     "by Illumina's GenomeStudio and Beadstudio analysis tools. The adpc.bin file is used as an input to " +
                     "<a href='https://genome.sph.umich.edu/wiki/VerifyIDintensity'>VerifyIDintensity</a> a tool for " +
-                    "detecting and estimating sample contamination of Illumina genotyping array data." +
+                    "detecting and estimating sample contamination of Illumina genotyping array data. " +
+                    "If more than one VCF is used, they must all have the same number of loci." +
                     "<h4>Usage example:</h4>" +
                     "<pre>" +
                     "java -jar picard.jar VcfToAdpc \\<br />" +
                     "      VCF=input.vcf \\<br />" +
-                    "      OUTPUT=output.adpc.bin" +
+                    "      OUTPUT=output.adpc.bin \\<br />" +
+                    "      SAMPLES_FILE=output.samples.txt \\<br />" +
+                    "      NUM_MARKERS_FILE=output.num_markers.txt \\<br />" +
                     "</pre>";
 
 
     private final Log log = Log.getInstance(VcfToAdpc.class);
 
-    @Argument(doc = "The Input VCF")
-    public File VCF;
+    @Argument(doc = "One or more VCF files containing array intensity data.")
+    public List<File> VCF;
 
     @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output (adpc.bin) file to write.")
     public File OUTPUT;
 
+    @Argument(shortName = "SF", doc = "A text file into which the names of the samples will be written. " +
+            "These will be in the same order as the data in the adpc.bin file.")
+    public File SAMPLES_FILE;
+
+    @Argument(shortName = "NMF", doc = "A text file into which the number of loci in the VCF will be written. " +
+            "This is useful for calling verifyIDIntensity.")
+    public File NUM_MARKERS_FILE;
+
     @Override
     protected int doWork() {
-        IOUtil.assertFileIsReadable(VCF);
+        final List<File> inputs = IOUtil.unrollFiles(VCF, IOUtil.VCF_EXTENSIONS);
+        IOUtil.assertFilesAreWritable(inputs);
+        IOUtil.assertFileIsWritable(SAMPLES_FILE);
```

Comment:
Should this also check the NUM_MARKERS_FILE?

Reply:
Yes.